packages/p2p/src/stores/general-store.js

```diff
@@ -36,6 +36,7 @@ export default class GeneralStore extends BaseStore {
     @observable review_period;
     @observable should_show_real_name = false;
     @observable should_show_popup = false;
+    @observable user_blocked_count = 0;
```

Comment:
Moved the block user count from my profile to general store since it already contains the subscription to` p2p_advertiser_info`, thus any updates to block user count is set to here